fuel-client/src/client/schema/tx.rs

```diff
@@ -175,22 +176,22 @@ pub enum TransactionStatus {
 #[derive(cynic::QueryFragment, Debug)]
 #[cynic(schema_path = "./assets/schema.sdl")]
 pub struct SubmittedStatus {
-    pub time: super::DateTime,
+    pub time: U64,
```

Comment:
It might be nice to keep a wrapper time around time so that is explicitly clear that the `u64` time is in TAI format and not unix time as commonly seen with `u64` time